package.json

```diff
@@ -42,6 +43,7 @@
         "build-enbr": "./build.js -l en,br",
         "build-enda": "./build.js -l en,da",
         "build-ende": "./build.js -l en,de",
+	"build-enes": "./build.js -l es,es",
```

Comment:
Typo here, you've got `es,es` instead of `en,es`.

And while you're doing that, could you also change the indentation to match the other lines (and on line 36 too)?